src/client/app/containers/UIOptionsContainer.js

```diff
@@ -23,7 +23,10 @@ function mapStateToProps(state) {
 function mapDispatchToProps(dispatch) {
 	return {
 		selectMeters: newSelectedMeterIDs => dispatch(changeSelectedMeters(newSelectedMeterIDs)),
-		fetchMetersDataIfNeeded: () => dispatch(fetchMetersDataIfNeeded())
```

Comment:
Fixed in next commit, good idea. Actions are complicated enough already, we don't need to make them purposely esoteric.